swarm/fuse/swarmfs_test.go

```diff
@@ -351,16 +344,18 @@ func (ta *testAPI) unmountWhenResourceBusy(t *testing.T) {
 
 	actualPath := filepath.Join(testMountDir, "2.txt")
```

Comment:
Good idea, there is a lot of cleanup needed on the Swarm codebase, it's on our list of things to do :wink:.